### DISCUSSION TITLE
Help requested

### DISCUSSION THREAD

**Discussion by cassiodorus** [plausible/analytics#996] [Anything Else]
I would greatly appreciate it if someone would look at a web page of mine and tell me if I am doing something particularly dumb. I can't seem to get Plausible to recognize any page hits from:

www.cassiodorus.com/schools        any of the pages

I am using Rstudio and rmarkdown to generate the pages, and then I insert what I think is the right script for Plausible and google (google fails too so I expect I am doing something outright wrong). I'd include the page html here but I think the context may be part of the problem.

**Comment by metmarkosaric** [comment]
hi @cassiodorus! had a quick look and the script seems to be loading all fine to me. do you see the data in your dashboard? you can [read more here](https://plausible.io/docs/troubleshoot-integration)

**Comment by alkarkhi** [comment]
Do you use an adblocker by any chance?
